### PR TITLE
Update receive page

### PR DESCRIPTION
**Pull request by el00ruobuob**
This include the receive page update, and the new merchant view.

### CODE REVIEW

**Comment by erciccione** [comment]
@el00ruobuob Could you rebase with latest master? I don't know why Travis failed here.

**Comment by el00ruobuob** [comment]
Rebased.  
Let's see how travis feels about it.

**Comment by el00ruobuob** [comment]
Good travis ;)

File: en/ch05.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
-The `Receive` tab provides tools for generating subaddresses, crafting payment requests, and monitoring incoming transactions.
+The `Receive` tab provides tools for generating subaddresses.
 
 (1) **Addresses:** This is a list of your primary address and subaddresses.  
 (2) **Create new address:** This button allows you to create new subaddresses. You can create as many as you would like. *Learn more about [subaddresses](https://monero.stackexchange.com/questions/3673/what-is-a-sub-address).*  
-(3) **Advanced options:** Clicking here you will be prompted to the advanced section.  
-(4) **Amount:** This is for creating a payment request, enter the amount of Monero you would like to receive.  
-(5) **Tracking:** If you tick this box, you'll see a list of incoming transactions.  
-(6) **QR code:** This is a QR code that has your selected address, and optionally the amount, embedded into it. It can be used as a way to give others your Monero address by scanning the code.  
+(3) **Set Label:** Click on this button to set this subaddresse label.
```
Comment (erciccione) [inline en/ch05.md]: subadresse

Comment (el00ruobuob) [reply en/ch05.md]: subaddress?

Comment (erciccione) [reply en/ch05.md]: Yep https://www.thefreedictionary.com/subaddress

**Pull request merged**